plugins/modules/vmware_object_custom_attributes_info.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, sky-joker
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: vmware_object_custom_attributes_info
+short_description: Gather custom attributes of an object
+author:
+  - sky-joker (@sky-joker)
+description:
+  - This module can be gathered custom attributes of an object.
+requirements:
+  - python >= 2.7
```

Comment:
Since Python 2 is EOL, we should start testing and using Python 3 only code. and set minimum requirement to Python 3.